### PR TITLE
Fix line length for IndexLifecycleActionIT

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
Relates #34884

### CODE REVIEW

File: server/src/test/java/org/elasticsearch/indexlifecycle/IndexLifecycleActionIT.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ public void testIndexLifecycleActionsWith11Shards1Backup() throws Exception {
         final String node1 = getLocalNodeId(server_1);
 
         logger.info("Creating index [test]");
-        CreateIndexResponse createIndexResponse = client().admin().indices().create(createIndexRequest("test").settings(settings)).actionGet();
+        CreateIndexResponse createIndexResponse = client().admin().indices().create(
```
Comment (pgomulka) [inline server/src/test/java/org/elasticsearch/indexlifecycle/IndexLifecycleActionIT.java]: I noticed that you break lines on '(' . Was that an intention? Reminds me of my early projects in Lisp :) I am worried that most of other refactoring would be breaking on chained method '.'
what do you think of breaking on '.'
for instance
```
client().admin().indices()
.create(reateIndexRequest("test").settings(settings)))
.actionGet();
```
or even one '.' per each line - although that might have been excessive

**Review by pgomulka — COMMENTED** [review]
looks good, but one comment

File: server/src/test/java/org/elasticsearch/indexlifecycle/IndexLifecycleActionIT.java (hunk 1 above)
Comment (danielmitterdorfer) [reply server/src/test/java/org/elasticsearch/indexlifecycle/IndexLifecycleActionIT.java]: The intention was to separate the request parameter from the actual call.

**Review by pgomulka — APPROVED** [review]
LGTM

**Comment by danielmitterdorfer** [comment]
Thank you for the review. :)

**Comment by danielmitterdorfer** [comment]
Backported to 6.x in https://github.com/elastic/elasticsearch/commit/7c82b15915affe7806a00be7a794435d941e65c1

**Pull request merged**